Mathlib/Algebra/Group/Units.lean

```diff
@@ -1172,10 +1183,15 @@ section NoncomputableDefs
 
 variable {M : Type*}
 
+-- Unused linter bug at `hM`
```

Comment:
Why do you need to name it, not just write `[Monoid M]`?

Reply:
Done